x-pack/plugins/maps/public/components/_index.scss

```diff
@@ -3,3 +3,19 @@
 @import './layer_panel/index';
 @import './widget_overlay/index';
 @import './toolbar_overlay/index';
+
+// The following code will only get compiled from this file
+
+// Custom SVG as background for zoom controls based off of EUI glyphs plusInCircleFilled and minusInCircleFilled
+// Also fixes dark mode
+.mapboxgl-ctrl-icon.mapboxgl-ctrl-zoom-in {
```

Comment:
Can we keep this in a separate `_hacks.scss` file like we normally do for this kind of thing?